src/components/texts/Texts.tsx

```diff
@@ -37,17 +37,18 @@ const Paragraph = styled.p<IText>`
 interface IPurpleText extends IText {
   underline: boolean;
   fontWeight: number;
+  cursor?: string;
   fontSize: "heading" | "subHeading" | "paragraph" | "subText" | "button";
 }
 
 const PurpleText = styled.p<IPurpleText>`
   line-height: 130%;
-  font-weight: ${props => props.fontWeight};
-  text-align: ${props => props.textAlign ?? "left"};
-  color: ${props => props.theme.colors.textSecondary};
-  text-decoration: ${props => (props.underline ? "underline" : "none")};
-  cursor: ${props => (props.underline ? "pointer" : "auto")};
-  font-size: ${props =>
+  font-weight: ${(props) => props.fontWeight};
+  text-align: ${(props) => props.textAlign ?? "left"};
+  color: ${(props) => props.theme.colors.textSecondary};
+  text-decoration: ${(props) => (props.underline ? "underline" : "none")};
```

Comment:
These two properties are be required...you can wrap the PurpleText in an anchor tag, then cursor will automatically be pointer and text-decoration can be set to none.